docs/site/LoopBack-3.x.md

```diff
@@ -37,80 +38,90 @@ In the persistence layer, users can contribute the following artifacts:
 At the public API side, users can define:
 
   1. Which built-in methods should be exposed (think of `disableRemoteMethodByName`)
-  1. Custom remote methods
-  2. before/after/afterError hooks at application-level
-  3. before/after/afterError hooks at model-level
-  4. before/after/afterError hooks at model method level
+  2. Custom remote methods
+  3. before/after/afterError hooks at application-level
+  4. before/after/afterError hooks at model-level
+  5. before/after/afterError hooks at model method level
 
-LoopBack Next was intentionally designed to allow users to choose their ORM/persistence solution, and our initial version of @loopback/repository is based on juggler 3.x. That makes it possible for users to reuse their existing model definitions, migrating their application incrementally.
+LoopBack 4 was intentionally designed to allow users to choose their own
+ORM/persistence solution, and our initial version of @loopback/repository
+is based on juggler 3.x. That makes it possible for users to reuse their
+existing model definitions, migrating their application incrementally.
+
+LoopBack 4 was intentionally designed to allow users to choose their own
```

Comment:
Duplicate of above paragrpah.

Reply:
Oops, forgot to get rid of the paragraph above when I copied it to edit it.